src/sagemaker/local/entities.py

```diff
@@ -38,6 +38,136 @@
 HEALTH_CHECK_TIMEOUT_LIMIT = 120
 
 
+class _LocalProcessingJob(object):
+    """Defines and starts a local processing job."""
+
+    _STARTING = "Starting"
+    _PROCESSING = "Processing"
+    _COMPLETED = "Completed"
+    _states = ["Starting", "Processing", "Completed"]
+
+    def __init__(self, container):
+        """
+        Args:
+            container: the local container object.
+        """
```

Comment:
nit: apply [pep-0257](https://www.python.org/dev/peps/pep-0257/#multi-line-docstrings) here and elsewhere (single summary line, blank line, optional multi-line body description, args, so forth)

```suggestion
        """Create a local container object.

        Args:
            container: the local container object.
        """
```

Reply:
Done